app/components/nft-web-view/nft-web-view.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+import { WebView, WebViewNavigation, WebViewProps } from "react-native-webview"
+
+import { COMMON_API_CONFIG } from "../../services/api/api-config"
+import { Linking } from "react-native"
+import { OnShouldStartLoadWithRequest } from "react-native-webview/lib/WebViewTypes"
+
+export function NFTWebView(props: WebViewProps) {
+
+  const handleShouldStartLoadWithRequest: OnShouldStartLoadWithRequest = ({ url }: WebViewNavigation) => {
+    if (/\.(pdf|epub)/i.test(url)) {
+      Linking.openURL(url)
+      return false
+    }
+    return true
+  }
+
+  return (
+    <WebView
+      {...props}
+      sharedCookiesEnabled={true}
+      decelerationRate={0.998}
+      // TODO: remove HACK after applicationNameForUserAgent type is fixed
+      {...{ applicationNameForUserAgent: COMMON_API_CONFIG.userAgent }}
+      onShouldStartLoadWithRequest={handleShouldStartLoadWithRequest}
```

Comment:
According to doc, `onShouldStartLoadWithRequest` is only supported in iOS but not android

Reply:
https://github.com/react-native-webview/react-native-webview/issues/1869#issuecomment-984578843
Actually I am not sure now, but seems it doesn't `just work` in android